### DISCUSSION TITLE
Bug: wrong border class on day cells

### DISCUSSION THREAD

**Discussion by muhammadsaeedparacha** [wdelfuego/nova-calendar#8] [Ideas & Feature Requests]
Removing the gray-800 border on calendar boxes significantly improves UI. 
When I opened the calendar the first time, so many black boxes was overwhelming and it looked far better in the picture in read.me

After removing the gray-800 border, the default rgb(229, 231, 235) is shown which blends in with the rest of the design like it's demonstrated in this picture:

<img width="1424" alt="Screenshot 2022-06-04 at 6 54 03 PM" src="https://user-images.githubusercontent.com/2680384/172003891-6f350e97-d3bb-4cfa-9963-4f1bc8568e7a.png">

**Comment by wdelfuego** [comment]
Hi and thanks for opening this issue!

The default calendar should look like this in light mode:
![light](https://user-images.githubusercontent.com/1326136/172017783-b28d5780-c84f-4744-aaa3-b88fada953f1.jpg)

And like this in dark mode:
![dark](https://user-images.githubusercontent.com/1326136/172017780-50fa7c30-de23-43ec-8d14-5f7bc97675a2.jpg)

This is in Safari 15.4 under MacOS Monterey 12.3.1.

Does the default calendar look different for anyone?
If so, could you post a screenshot and tell me which operating system + version and browser + version you're using?

**Comment by wdelfuego** [comment]
Your fonts also look different. Are you sure it's not a custom css issue?
What version of Nova are you using and did you publish its assets correctly?

**Comment by muhammadsaeedparacha** [comment]
So the red color is because of custom styling through nova config file for Nova Branding. I removed it and opened in safari (previously I was using Chrome, latest version) and the borders are still there, like:

<img width="1323" alt="Screenshot 2022-06-05 at 12 16 12 AM" src="https://user-images.githubusercontent.com/2680384/172022465-a6265006-0a1b-4eae-8c4f-6eb8dbfa46ee.png">

I will check this on a fresh install and report back

**Comment by muhammadsaeedparacha** [comment]
@wdelfuego 
I think the problem is, the component has `border-gray-800`, and it might not be showing up for you because nova implements tailwind treeshaking and the class `border-gray-800` doesn't even exist in nova version of tailwind compiled css.

I had manually imported a complete version of tailwind, so that all tailwind classes would exist
After I removed my own version of complete tailwindcss, it fixes it. 

Fixed:
<img width="1721" alt="Screenshot 2022-06-05 at 12 34 42 AM" src="https://user-images.githubusercontent.com/2680384/172023169-6d431efa-9c25-4bbc-a5e7-4b0a3c098ca8.png">

Solution:
remove the `border-gray-800` class from `tool.vue`

**Comment by muhammadsaeedparacha** [comment]
I'm sending a PR for this

**Comment by wdelfuego** [comment]
Good catch @muhammadsaeedparacha, thanks! 👍

**Comment by wdelfuego** [comment]
This issue resurfaced recently but should be fixed once and for all as of 1.5.1.
